pkg/3scale/amp/component/s3.go

```diff
@@ -197,7 +195,7 @@ func (s3 *S3) addS3PostprocessOptionsToSystemEnvironmentCfgMap(objects []runtime
 		}
 	}
 
-	systemEnvCfgMap.Data["FILE_UPLOAD_STORAGE"] = s3.Options.fileUploadStorage
+	systemEnvCfgMap.Data["FILE_UPLOAD_STORAGE"] = "s3"
```

Comment:
I think this line can be removed.

Reply:
hmm I don't think so because otherwise the element would not be created in the ConfigMap and the envvar could not be defined. Also the default would be local disk which we don't want in that context

Reply:
Oh, I misunderstood the code. I thought this was for template parameter (which got removed). My bad.